azerothcore/azerothcore-wotlk
# refactor(Scripts/Commands): Update cs_quest to the new model

## Changes Proposed:
-  Update cs_quest to the new parsing model
-  Allow all commands to be used from console and to target offline players

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Tested in-game.
- 


## How to Test the Changes:


1. quest add 102 $charname
2. quest complete 102 $charname
3. Log in, check if the quest is complete

...
Test using the commands when characters are online/offline, from console and in-game.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
